13-loops/exercises/03-sum-up-to-n/main.go

```diff
@@ -36,4 +34,33 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	if len(os.Args)-1 != 2 {
+		fmt.Println("Please pass min and max integers.")
+		return
+	}
+
+	var sum int
+
+	min, errMin := strconv.Atoi(os.Args[1])
```

Comment:
I think it is better to check for one error at a time; it appears clean

Reply:
Fixed